### PR TITLE
Polls server for proposal credit payment updates

### PR DESCRIPTION
**Pull request by thi4go**
Proposal payment action triggers a polling update for the payment until confirmation hits minimum (two). 

https://github.com/decred/politeiagui/issues/586

### CODE REVIEW

**Comment by thi4go** [comment]
Note: plan to add the confirmation status on the notifications UI once it goes in 👍

File: src/lib/local_storage.js [hunk 1]
```diff
@@ -81,7 +81,22 @@ const handleSaveAppDraftProposals = (state) => {
...
+  saveStateLocalStorage(
+    set(stateFromLs, [ "app", "paymentPollingQueue" ], queueFromStore),
+    email
+  );
+};
+
 export const handleSaveStateToLocalStorage = (state) => {
   handleSaveApiMe(state);
   handleSaveAppDraftProposals(state);
+  handlePaymentPollingQueue(state);
```
Comment (fernandoabolafio) [inline src/lib/local_storage.js]: Please use `handleAppPaymentPollingQueue` for naming consistency.

**Comment by thi4go** [comment]
@fernandoabolafio great suggestion! Didn't really know where to put the confirmation msg, but it will do good on the table. Will set this up and commit the changes

**Comment by thi4go** [comment]
Hey @fernandoabolafio, did some changes on the table ui, think it's more user friendly now. Also change some minor things like button alignments on account page.

New table:
image

**Comment by tiagoalvesdulce** [comment]
Tested and works great.

**Pull request merged**